RDKShell/RDKShell.cpp

```diff
@@ -1119,7 +1138,11 @@ namespace WPEFramework {
             m_timer.setInterval(RECONNECTION_TIME_IN_MILLISECONDS);
             m_timer.start();
             std::cout << "Started SystemServices connection timer" << std::endl;
-
+            char* rdkshelltype = getenv("RDKSHELL_COMPOSITOR_TYPE");
+            if (strcmp(rdkshelltype , "surface") == 0)
```

Comment:
have a null check for rdkshelltype as below:
if ((NULL != rdkshelltype ) && (strcmp(rdkshelltype , "surface") == 0))